test/package/test_misc.py

```diff
@@ -139,31 +137,6 @@ def test_is_from_package(self):
         self.assertFalse(is_from_package(obj))
         self.assertTrue(is_from_package(loaded_obj))
 
-
-    @skipIf(version_info < (3, 7), "mock uses __getattr__ a 3.7 feature")
```

Comment:
I am deleting this test because I want to remove the `require_module` extension point, which has a similar "may bust our invariants" issue. I think `require_module` was more useful before we had robust pattern matching.